documentation-website/Writerside/topics/Database-and-DataSource.md

```diff
@@ -0,0 +1,130 @@
+```
+* Oracle
+```kotlin
+Database.connect("jdbc:oracle:thin:@//localhost:1521/test", driver = "oracle.jdbc.OracleDriver", 
+                 user = "root", password = "your_pwd")  
+// Gradle
+// Oracle jdbc-driver should be obtained from Oracle maven repo: https://blogs.oracle.com/dev2dev/get-oracle-jdbc-drivers-and-ucp-from-oracle-maven-repository-without-ides
+```
+* SQLite
+```kotlin
+// In file
+Database.connect("jdbc:sqlite:/data/data.db", "org.sqlite.JDBC")  
+// In memory
+Database.connect("jdbc:sqlite:file:test?mode=memory&cache=shared", "org.sqlite.JDBC")  
+// For both: set SQLite compatible isolation level, see 
+// https://github.com/JetBrains/Exposed/wiki/FAQ
+TransactionManager.manager.defaultIsolationLevel = 
+    Connection.TRANSACTION_SERIALIZABLE
+    // or Connection.TRANSACTION_READ_UNCOMMITTED
+// Gradle
+implementation("org.xerial:sqlite-jdbc:3.44.1.0")  
+```  
+* H2
+```kotlin
+// Database in file, needs full path or relative path starting with ./
+Database.connect("jdbc:h2:./myh2file", "org.h2.Driver")
+// In memory
+Database.connect("jdbc:h2:mem:regular", "org.h2.Driver")  
+// In memory / keep alive between connections/transactions
+Database.connect("jdbc:h2:mem:regular;DB_CLOSE_DELAY=-1;", "org.h2.Driver")  
+// Gradle
+implementation("com.h2database:h2:2.2.224")  
+```  
+* SQL Server
+```kotlin
+Database.connect("jdbc:sqlserver://localhost:32768;databaseName=test", "com.microsoft.sqlserver.jdbc.SQLServerDriver", 
+                 user = "root", password = "your_pwd")  
+// Gradle
+implementation("com.microsoft.sqlserver:mssql-jdbc:9.4.1.jre8")  
+```
+
+### HikariCP
+To use a JDBC connection pool like [HikariCP](https://github.com/brettwooldridge/HikariCP), first set up a `HikariConfig` class.
+This example uses the MySQL JDBC driver (see the official reference for [MySQL configuration](https://github.com/brettwooldridge/HikariCP/wiki/MySQL-Configuration) details):
+```kotlin
+val config = HikariConfig().apply {
+    jdbcUrl = "jdbc:mysql://localhost/dbname"
+    driverClassName = "com.mysql.cj.jdbc.Driver"
+    username = "username"
+    password = "password"
+    maximumPoolSize = 6
+    // as of version 0.46.0, if these options are set here, they do not need to be duplicated in DatabaseConfig
+    isReadOnly = false
+    transactionIsolation = "TRANSACTION_SERIALIZABLE"
+}
+
+// Gradle
+implementation "mysql:mysql-connector-java:8.0.33"
+implementation "com.zaxxer:HikariCP:4.0.3"
+```
+Then instantiate a `HikariDataSource` with this configuration class and provide it to `Database.connect()`:
+```kotlin
+val dataSource = HikariDataSource(config)
+
+Database.connect(
+    datasource = dataSource,
+    databaseConfig = DatabaseConfig {
+        // set other parameters here
+    }
+)
+```
+**Note: ** Since version 0.46.0, when configured directly in the `HikariConfig` class,
+values like `transactionIsolation` and `isReadOnly` will be used by Exposed when creating transactions.
```

Comment:
Leaving the space after the colon leads to plain text without formatting: "**Note: **".
Could the word be bold instead or the entire info be placed in a < note > block?

Reply:
replaced with <note> block